website/versioned_docs/version-v1.2/cli.md

```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
+| `setVerifyingKeys`   | Set the verifying keys                                             | `-s, --state-tree-depth <stateTreeDepth>`: The state tree depth <br/> `-i, --int-state-tree-depth <intStateTreeDepth>`: The intermediate state tree depth <br/> `-m, --msg-tree-depth <messageTreeDepth>`: The message tree depth <br/> `-v, --vote-option-tree-depth <voteOptionTreeDepth>`: The vote option tree depth <br/> `-b, --msg-batch-depth <messageBatchDepth>`: The message batch depth <br/> `-pqv, --process-messages-zkey-qv <processMessagesZkeyPathQv>`: The process messages qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-tqv, --tally-votes-zkey-qv <tallyVotesZkeyPathQv>`: The tally votes qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-pnqv, --process-messages-zkey-non-qv <processMessagesZkeyPathNonQv>`: The process messages non-qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-tnqv, --tally-votes-zkey-non-qv <tallyVotesZkeyPathNonQv>`: The tally votes qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-uq, --use-quadratic-voting"`: Whether to use quadratic voting <br/> `-k, --vk-registry <vkRegistry>`: The vk registry contract address <br/> `-q, --quiet`: Whether to print values to the console <br/> `-r, --rpc-provider <provider>`: The rpc provider URL              |
```

Comment:
`-pqv`, `-tqv`, `-pnqv`, `-tnqv` flags might be in the current version only? 

missing following command:
`-ss, --subsidy-zkey <subsidyZkeyPath>`
    the subsidy zkey path (see different options for zkey files to use specific circuits https://maci.pse.dev/docs/trusted-setup, https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)

Reply:
Same here, the verifying keys on v 1.2.0 has these params 
![image](https://github.com/privacy-scaling-explorations/maci/assets/578688/d1a25291-e47a-48dd-bd30-a4d4edc413e1)